app/enterprise/2.2.x/support-policy.md

```diff
@@ -3,37 +3,73 @@ title: Kong Enterprise Version Support
 toc: false
 ---
 
+The support for Kong Enterprise software versions is explained in this topic.
 
-The support for Kong Enterprise software versions is explained in this topic. 
+## Types of Releases
+Kong primarily follows [semantic versioning](https://semver.org/) (SemVer) with its products. That is, products typically follow a pattern of `<MAJOR_VERSION>.<MINOR_VERSION>.<PATCH_VERSION>`. Kong Enterprise additionally has one more decimal point on the right which identifies a sub-patch based on the Kong Community Gateway. For the purposes of this support document:
 
-For customers with an active enterprise support agreement, Kong provides full support for each release for 12 months from the released date. 
+* **Major Version** refers to a version of Kong Enterprise identified by the number to the left of the leftmost decimal point. For example, 2.1.3.0 has Major Version 2 and 1.3.0.0 has Major Version 1.
 
-Full support includes:
+* **Minor Version** means a version of the Software identified by a change in the number in between the two leftmost decimal points (x.X.x). For example, 2.1.3.0 has Minor Version 1 and 1.3.0.0 has Minor Version 3.
 
-* Technical support on the use of the software as documented, assistance with application configuration, and guidelines on performance tuning. 
+* The **First Major Version Release** is the first chronologically generally available release of a given Major Version. The First Major Version Release of Major Version 2 of Kong Enterprise was 2.1.3.0 and the First Major Version Release of Major Version 1 was 1.3.0.0.
 
-* Access to all error corrections, bug fixes, software updates, and software upgrades to the Software that Kong makes generally available to other Kong enterprise customers who have purchased a license to the same Software purchased by Customer. 
+* The **Latest Major/Minor Version Release** is the most recent chronologically generally available Minor Version release of a given Major Version.
 
-* Development of workarounds or patches for customers upon request, and where technically possible, for the following: Urgent issues; Critical and High Security Vulnerabilities; High priority issues where the product in a non-production environment is nearing a critical milestone, is unusable, and no workaround is available.
+For customers with an active enterprise support agreement, beginning with Kong Enterprise release version 2.1.3.x<sup>*</sup>. Kong provides support for each Major Version release of Kong Enterprise for 24 months from the date of the First Major Version Release.
```

Comment:
The last sentence here should read "Kong provides support for each Major Version release of Kong Enterprise for 24 months from the release date of the Minor Version."